.zenodo.json

```diff
@@ -271,6 +271,10 @@
       "name": "Blair, Ross W.",
       "orcid": "0000-0003-3007-1056"
     },
+    {
+      "affiliation": "Department of Radiology, University Hospital of Lausanne and University of Lausanne",
+      "name": "Provins, Céline",
```

Comment:
Please, add yourself to `.maint/contributors.json` instead.

Other than that, this looks great 👍 thanks!

Reply:
Ok. Should I remove myself from the zenodo.json as well ?
It might be worth changing the pull request template, because I added myself in the zenodo.json following the indications for new contributors written in the pull request template.

Reply:
Yes, that would be cleaner. Before every release, we update the zenodo file pulling from the jsons under `.maint/` so your name will be inserted at that time.